includes/import/class-wc-product-csv-importer.php

```diff
@@ -39,6 +39,8 @@ public function __construct( $file, $params = array() ) {
 			'update_existing'  => false, // Whether to update existing items.
 			'delimiter'        => ',', // CSV delimiter.
 			'prevent_timeouts' => true, // Check memory and time usage and abort if reaching limit.
+			'enclosure'        => '"', //The enclousure argument passed to fgetcsv.
+			'escape'           => '\\', //The escape argument passed to fgetcsv.
```

Comment:
The comments here could be more clear than just saying that are passing to `fgetcsv`, or don't need any comment.